### ISSUE TITLE
How to extend uws?

### ISSUE THREAD

**Issue by Influxx** [alexhultman/uWebSockets#142]
With `ws` it is possible to do:

```
const WebSocket = require('ws');

WebSocket.prototype.sendPacket = function(packet) {
  if (this.readyState == WebSocket.OPEN && (this._socket.bufferSize == 0)) {
      try {
          this.send(packet.build(), {binary: true});
      } catch (e) {
          console.log('Packet error');
      }
  } else {
      this.emit('close');
      this.removeAllListeners();
  }
};
```
How would one implement this with `uws`?

**Comment by alexhultman** [comment]
This lib is a server only, you are trying to extend a client

**Comment by alexhultman** [comment]
This has been fixed in 0.7.2. Apologies for the misunderstandings.
